index.d.ts

```diff
@@ -0,0 +1,31 @@
+export = UserAgentBag;
+export as namespace UserAgentBag;
+
+declare namespace UserAgentBag {
+  export type Entry = [string, string | null];
+
+  export type Entries = Iterable<Entry>;
+
+  type UserAgentBagNode = (
```

Comment:
Because this is never used by consumers, is this needed? (If not, we might also want to remove it from the Flow types in a separate PR—not sure why that's there, either...)

Reply:
Ah, I just blindly converted everything in the flow type definition file. Removed.